pkg/cli/iam_cleanup.go

```diff
@@ -0,0 +1,178 @@
+
+      {{ COMMAND }} -path "/path/to/file.yaml"
+
+Cleanup of the IAM request YAML file and output applied IAM changes:
+
+      {{ COMMAND }} -path "/path/to/file.yaml" -verbose
+`
+}
+
+func (c *IAMCleanupCommand) Flags() *cli.FlagSet {
+	set := c.NewFlagSet()
+
+	// Command options
+	f := set.NewSection("COMMAND OPTIONS")
+
+	f.StringVar(&cli.StringVar{
+		Name:    "path",
+		Target:  &c.flagPath,
+		Example: "/path/to/file.yaml",
+		Predict: predict.Files("*"),
+		Usage:   `The path of IAM request file, in YAML format.`,
+	})
+
+	f.BoolVar(&cli.BoolVar{
+		Name:    "verbose",
+		Target:  &c.flagVerbose,
+		Default: false,
+		Usage:   `Turn on verbose mode to print updated IAM policies. Note that it may contain sensitive information`,
+	})
+
+	return set
+}
+
+func (c *IAMCleanupCommand) Run(ctx context.Context, args []string) error {
+	f := c.Flags()
+	if err := f.Parse(args); err != nil {
+		return fmt.Errorf("failed to parse flags: %w", err)
+	}
+	args = f.Args()
+	if len(args) > 0 {
+		return fmt.Errorf("unexpected arguments: %q", args)
+	}
+
+	if c.flagPath == "" {
+		return fmt.Errorf("path is required")
+	}
+
+	return c.cleanupIAM(ctx)
+}
+
+func (c *IAMCleanupCommand) cleanupIAM(ctx context.Context) error {
+	// Read request from file path.
+	var req v1alpha1.IAMRequest
+	if err := requestutil.ReadRequestFromPath(c.flagPath, &req); err != nil {
+		return fmt.Errorf("failed to read %T: %w", &req, err)
+	}
+
+	if err := v1alpha1.ValidateIAMRequest(&req); err != nil {
+		return fmt.Errorf("failed to validate %T: %w", &req, err)
+	}
+
+	var h iamCleanupHandler
+	if c.testHandler != nil {
+		// Use testHandler if it is for testing.
+		h = c.testHandler
+	} else {
+		// Create resource manager clients.
+		organizationsClient, err := resourcemanager.NewOrganizationsClient(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to create organizations client: %w", err)
+		}
+		defer organizationsClient.Close()
+
+		foldersClient, err := resourcemanager.NewFoldersClient(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to create folders client: %w", err)
+		}
+		defer foldersClient.Close()
+
+		projectsClient, err := resourcemanager.NewProjectsClient(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to create projects client: %w", err)
+		}
+		defer projectsClient.Close()
+
+		// Create IAMHandler with the clients.
+		h, err = handler.NewIAMHandler(
+			ctx,
+			organizationsClient,
+			foldersClient,
+			projectsClient,
+		)
+		if err != nil {
+			return fmt.Errorf("failed to create IAM handler: %w", err)
+		}
+	}
+
+	resp, err := h.Cleanup(ctx, &req)
+	// The error here might only be errrors of parsing the condition expiration
+	// expression of some IAM bindings, it does not necessarily mean that it
+	// failed to cleanup the requested bindings.
+	if err != nil {
+		return fmt.Errorf("encountered error when cleanning up IAM policy: %w", err)
+	}
+
+	printHeader(c.Stdout(), "Successfully Removed Requested Bindings")
```

Comment:
```suggestion
	printHeader(c.Stdout(), "Successfully removed the requested bindings.")
```

Reply:
in header? `------Successfully Removed Requested Bindings------` is better than `------Successfully removed the requested bindings.------` I think